actions/get-changed-files/action.yml

```diff
@@ -16,7 +19,10 @@ runs:
       with:
         script: |
           try {
-            require('./actions/get-changed-files/index.js')({github, context, core}).catch(err => core.setFailed(err.message))
+            require('./actions/get-changed-files/index.js')({
+              github, context, core,
+              directoriesRaw: `${{ inputs.directories }}`,
```

Comment:
If `inputs.directories` isn't provided, this will just be an empty string, right?

Reply:
the default value is `.github/`

Reply:
oh wait jk I was thinking about a different option. Yes, it will be the empty string.